.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       matrix:
         os: [ubuntu-latest, macOS-latest, windows-latest]
         cabal: ["3.6.2.0"]
```

Comment:
Let's also update `cabal` to `3.8.1.0` everywhere

Reply:
Updated